core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -4489,17 +4488,11 @@ public static void registerGauges(final UfsManager ufsManager, final InodeTree i
       MetricsSystem.registerGaugeIfAbsent(MetricKey.MASTER_FILES_PINNED.getName(),
           inodeTree::getPinnedSize);
       MetricsSystem.registerGaugeIfAbsent(MetricKey.MASTER_FILES_TO_PERSIST.getName(),
-          () -> inodeTree.getToBePersistedIds().size());
+          inodeTree.getToBePersistedIds()::size);
       MetricsSystem.registerGaugeIfAbsent(MetricKey.MASTER_TOTAL_PATHS.getName(),
           inodeTree::getInodeCount);
       MetricsSystem.registerGaugeIfAbsent(MetricKey.MASTER_FILE_SIZE.getName(),
-          () -> StreamSupport.stream(
-              inodeTree.getFileSizeHistogram().logarithmicBucketValues(1024, 1024).spliterator(),
-              false)
-              .map(x -> new Pair<>(
-                  new Pair<>(x.getDoubleValueIteratedFrom(), x.getDoubleValueIteratedTo()),
-                  x.getCountAddedInThisIterationStep()))
-              .collect(Collectors.toMap(Pair::getFirst, Pair::getSecond)));
+          inodeTree::getFileSizeHistogram);
```

Comment:
Could you share an example metric output?

Reply:
```
value: {
1024: 4,
1048576: 1,
1073741824: 1,
1099511627776: 0,
9223372036854775807: 0
}```